### PR TITLE
A few games for whitelisting

### PR DESCRIPTION
**Pull request by daddeltrotter**

### CODE REVIEW

**Comment by Samsagax** [comment]
Could this be added in the https://github.com/gamer-os/steam-buddy repo?

File: steam-tweaks.yaml [hunk 1]
```diff
@@ -1042,21 +1060,73 @@
...
 
+"3920": # Sid Meier's Pirates!
+  compat_tool: proton_63
+
+"7600": # Sid Meier's Railroads
+  compat_tool: proton_513
+
+"282210": # Sid Meier's Starships
+  compat_tool: proton_63
+  launch_options: rm Starships64.exe && ln -s Starships.exe Starships64.exe; %command%
```
Comment (Samsagax) [inline steam-tweaks.yaml:1116]: Shouldn't this be a patch? Or the game recreates it everytime?

Comment (alkazar) [reply steam-tweaks.yaml:1116]: If there is an update or the user runs "verify files", the original file will be restored.

Patches are only really needed if you need something to run before the the "first time setup" step.

Comment (Samsagax) [reply steam-tweaks.yaml:1116]: Thanks for clarification

**Comment by daddeltrotter** [comment]
I wasn't aware that you want to merge Steam-tweaks into steam-buddy.
So wherever you want tme to do the changes I will do.:)

**Comment by Samsagax** [comment]
> I wasn't aware that you want to merge Steam-tweaks into steam-buddy.
> So wherever you want tme to do the changes I will do.:)

Yeah. As per [this commit](https://github.com/gamer-os/steam-buddy/commit/102d127b966436280a36be4e398ec4861c9593be) onward tweaks is merged into buddy.

**Pull request closed without merging**